internal/api/peer/delegatedpull.go

```diff
@@ -56,7 +56,8 @@ func (a *API) handleDelegatedPullManifest(w http.ResponseWriter, r *http.Request
 		Password: r.Header.Get("X-Keppel-Delegated-Pull-Password"), //may be empty
 	}
 	ref := models.ParseManifestReference(vars["reference"])
-	manifestBytes, manifestMediaType, err := rc.DownloadManifest(ref, &opts)
+	manifestBytes, manifestMediaType, err := rc.DownloadManifest(r.Context(), ref, &opts)
+	//nolint:errorlint // not applicable
```

Comment:
This description is incredibly vague. In Limes, I changed these to
```suggestion
	//nolint:errorlint // a type cast is clearer than errors.As()
```

Same for all the other identical nolint comments.